notifications.json

```diff
@@ -0,0 +1,478 @@
+            "in": "query",
+            "type": "integer",
+            "default": 1,
+            "minimum": 1
+        },
+        "trait:pagination:page_size": {
+            "name": "page_size",
+            "in": "query",
+            "type": "integer",
+            "default": 50,
+            "minimum": 1
+        }
+    },
+    "responses": {
+        "trait:apiErrors:400": {
+            "description": "",
+            "schema": {
+                "description": "Indicates that the request could not be parsed as JSON, or it contained invalid fields.",
+                "type": "object",
+                "properties": {
+                    "error": {
+                        "type": "string",
+                        "default": "BadRequestError"
+                    },
+                    "message": {
+                        "type": "string",
+                        "default": ""
+                    },
+                    "status": {
+                        "type": "integer",
+                        "default": 400
+                    }
+                }
+            }
+        },
+        "trait:apiErrors:401": {
+            "description": "",
+            "schema": {
+                "description": "There was an error authenticating the sender account.",
+                "type": "object",
+                "properties": {
+                    "error": {
+                        "type": "string",
+                        "default": "Authorisation header is missing"
+                    },
+                    "message": {
+                        "type": "string"
+                    },
+                    "status": {
+                        "type": "integer",
+                        "default": 401
+                    }
+                }
+            }
+        },
+        "trait:apiErrors:403": {
+            "description": "",
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "error": {
+                        "type": "string",
+                        "default": "Forbidden"
+                    },
+                    "message": {
+                        "type": "string"
+                    },
+                    "status": {
+                        "type": "integer",
+                        "default": 403
+                    }
+                }
+            }
+        },
+        "trait:apiErrors:404": {
+            "description": "",
+            "schema": {
+                "type": "object",
+                "description": "Object Not Found.",
+                "properties": {
+                    "error": {
+                        "type": "string"
+                    },
+                    "message": {
+                        "type": "string"
+                    },
+                    "status": {
+                        "type": "integer"
+                    }
+                }
+            }
+        },
+        "trait:apiErrors:429": {
```

Comment:
let's leave this out for now, we haven't defined the specs of how quota should work yet